### DISCUSSION TITLE
Disease Ontology: Mapping of ICD10 codes to DOID

### DISCUSSION THREAD

**Discussion by andaldanar** [DiseaseOntology/HumanDiseaseOntology#1044] [Q&A] [answered]
I recently discovered Disease Ontology, and I have found the mapping files in [HumanDiseaseOntology/src/DOreports/](https://github.com/DiseaseOntology/HumanDiseaseOntology/tree/main/src/DOreports) incredibly useful - thank you for such great work!

I would like to know if there is a file/table/CSV that has mappings of ICD10 codes to DOID? I’m not that familiar with many of the filetypes in the repo, but I was wondering if such a table exists as the ones in the DOreports.

Thank you for your help!

Andres

**Comment by allenbaron** [comment, marked as answer]
Hi Andres,
We're glad to hear the Disease Ontology is useful to you. We'd love to hear more about your use case.

The mapping files in DOreports are not regularly updated. They were generated for a test we did a few months ago. I've created an up-to-date dump of all the DOID mappings to external sources (DO's xrefs), including ICD-10 codes (download: [XREFSinDO_v2022-06-07.csv](https://github.com/DiseaseOntology/HumanDiseaseOntology/files/8872754/XREFSinDO_v2022-06-07.csv)). The data correspond to the latest DO release: [v2022-06-07](https://github.com/DiseaseOntology/HumanDiseaseOntology/releases/tag/v2022-06-07) and were extracted directly from the doid.owl file using the SPARQL query language.

Let us know if this meets your needs and if there's anything else we can help with.

Best,
Allen

**Reply by andaldanar** [reply]
Hi Allen,

I'm trying to map clinical trials (AACT database) to their ICD10 codes using MeSH terms.

Thank you so much for the file - it is very helpful!

I just added to the list learning to query owl files using SPARQL :)

Best regards,
Andres

**Comment by allenbaron** [comment]
For anyone looking for a **TSV** of mappings/cross-references (xrefs) between diseases in the Human Disease Ontology and other clinical vocabularies, an `allXREFinDO.tsv` file is now generated with each data release (along with a few vocabulary specific files) and can be found in the [DOreports](https://github.com/DiseaseOntology/HumanDiseaseOntology/tree/main/DOreports) directory.
